packages/did/src/Did.chain.ts

```diff
@@ -571,3 +571,9 @@ export function encodeDidSignature(
     [alg]: hexToU8a(signature.signature),
   }
 }
+
+export interface Web3NameOwner extends Struct {
```

Comment:
M: If we have a web3 chain file, I think this declaration would rather be better there than here.

Reply:
fixed